.gitignore

```diff
@@ -3,3 +3,5 @@
 # For normal case, output file should not be ignored, and it acts as
 # a signal that our testcase need to be fixed.
 examples/bad-case/simple/select.output
+# Ignoring cargo.lock file
+Cargo.lock
```

Comment:
I'm afraid this is not enough, you also need to delete Cargo.lock from git, 
```
git rm Cargo.lock
```
And remove lock check in our CI
- https://github.com/ceresdb/sqlness/blob/main/.github/workflows/ci.yml#L51
- https://github.com/ceresdb/sqlness/blob/main/.github/workflows/ci.yml#L85

Reply:
> I'm afraid this is not enough, you also need to delete Cargo.lock from git,
> 
> ```
> git rm Cargo.lock
> ```
> 
> And remove lock check in our CI
> 
> * https://github.com/ceresdb/sqlness/blob/main/.github/workflows/ci.yml#L51
> * https://github.com/ceresdb/sqlness/blob/main/.github/workflows/ci.yml#L85

Done